opennlp-docs/src/docbkx/model-loading.xml

```diff
@@ -0,0 +1,147 @@
+		<title>Loading a pre-trained OpenNLP model from the classpath</title>
+        <para>
+            First, you need to add the following dependency to your classpath:
+
+            <programlisting language="xml">
+                <![CDATA[
+<dependency>
+  <groupId>org.apache.opennlp</groupId>
+  <artifactId>opennlp-tools-models</artifactId>
+  <version>CURRENT_OPENNLP_VERSION</version>
+</dependency>
+]]>
+            </programlisting>
+
+            by using our pre-trained models or by building custom models as described later in this chapter.
+            If you need advanced classpath scanning capabilities, you should also add the classgraph library to your classpath.
+
+            <programlisting language="xml">
+                <![CDATA[
+<dependency>
+    <groupId>io.github.classgraph</groupId>
+    <artifactId>classgraph</artifactId>
+    <version>CURRENT_CLASSGRAPH_VERSION</version>
+</dependency>
+]]>
+            </programlisting>
+
+            Make sure you replace the placeholders with the appropriate version values.
+
+            Next, you can search for such a model and load it from the classpath:
+
+            <programlisting language="java">
+                <![CDATA[
+final ClassgraphModelFinder finder = new ClassgraphModelFinder(); // or use new SimpleClassPathModelFinder()
+final ClassPathModelLoader loader = new ClassPathModelLoader();
+final Set<ClassPathModelEntry> models = finder.findModels(false);
+for(ClassPathModelEntry entry : models) {
+
+   final ClassPathModel model = loader.load(model);
+
+   if(model != null) {
+      System.out.println(model.name());
+      System.out.println(model.sha256());
+      System.out.println(model.version());
+      System.out.println(model.language());
+      // do something with the model by consuming the byte array
+   }
+}]]>
+            </programlisting>
+
+        </para>
+	</section>
+
+
+	<section id="tools.model.bundle">
+		<title>Bundling a custom trained OpenNLP model for the classpath</title>
+		<para>
+            If you intend to provide your own custom trained OpenNLP models as JAR files for classpath discovery,
+            we recommend that you have a look at our setup in the <ulink url="https://github.com/apache/opennlp-models">OpenNLP Models
+            repository</ulink>. We recommend to put one model per JAR file.
+
+            Make sure you add a model.properties file with the following content
+
+       <programlisting language="java">
+                <![CDATA[
+model.name=${model.name}
+model.version=${model.version}
+model.sha256=${model.sha256}
+model.language=${model.language}
+]]>
+            </programlisting>
+
+            Make sure to replace the values accordingly and configure your build tool to include the binary model and the model.properties
+            in the resulting JAR file.
+
+            To load such a custom model, you may need to adjust the pattern for classpath scanning, i.e. if you name your model "custom-opennlp-model",
```

Comment:
Better avoid "i.e." in non-academic text auch as a dev manual. Better: "that is"